src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -224,6 +249,104 @@ def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
         return record
 
 
+class BertScoreWithDelimiter(Transform):
```

Comment:
I don't think we should create a separate transform for this; there's far too much overlap between this and the existing `BertScore`. Can you think of a way to compute the inputs to be fed to `BertScore` and then rely on calling `BertScore`? You will probably need a new transform for preprocessing and post-processing (i.e. figure out the inputs to `BertScore` and then aggregate the outputs of the multiple bert scores). When designing this component, try to keep it as generic as possible, so ideally don't restrict the use case just to splitting target outputs by a delimiter, for example.